src/Data/Sum/Relation/Binary/LeftOrder.agda

```diff
@@ -78,6 +79,17 @@ module _ {a₁ a₂} {A₁ : Set a₁} {A₂ : Set a₂}
   ⊎-<-decidable dec₁ dec₂ (inj₂ x) (inj₁ y) = no λ()
   ⊎-<-decidable dec₁ dec₂ (inj₂ x) (inj₂ y) = Dec.map′ ₂∼₂ drop-inj₂ (dec₂ x y)
 
+  ⊎-<-WellFounded : WellFounded ∼₁ → WellFounded ∼₂ → WellFounded (∼₁ ⊎-< ∼₂)
+  ⊎-<-WellFounded wf₁ wf₂ x = acc (⊎-<-acc x)
+    where ⊎-<-acc₁ : ∀ {x} → Acc ∼₁ x → WfRec (∼₁ ⊎-< ∼₂) (Acc (∼₁ ⊎-< ∼₂)) (inj₁ x)
+          ⊎-<-acc₁ (acc rec) (inj₁ y) (₁∼₁ x∼₁y) = acc (⊎-<-acc₁ (rec y x∼₁y))
```

Comment:
Thanks! Minor formatting changes only:
1. Could we have them at the same indentation as the `where`?
2. Could we have spaces in between the individual lemmas?
3. `WellFounded` in the name should be `wellFounded`